### DISCUSSION TITLE
Synchronize whitelist domains using google sync

### DISCUSSION THREAD

**Discussion by WimmoX** [Cookie-AutoDelete/Cookie-AutoDelete#1090] [Ideas / Feature Requests] [answered]
Hello, would it be possible to synchronize the white lists across my computers using the Chrome sync feature? That would save me a lot of clicks, as I work on various computers at home and at work.

**Comment by kennethtran93** [comment, marked as answer]
Duplicate of #30.

**Comment by conuaunoc** [comment]
I think even better than using built-in browser sync would be Google Drive/Dropbox/OneDrive sync, which would work across browsers. The uBlacklist extension does this and it's great.
